app/screens/custom_status/custom_status_modal.tsx

```diff
@@ -289,8 +258,8 @@ class CustomStatusModal extends PureComponent<Props, State> {
                 >
                     <View style={style.scrollView}>
                         {customStatusInput}
-                        {this.renderRecentCustomStatuses()}
-                        {this.renderCustomStatusSuggestions()}
+                        {this.renderSuggestions('recent')}
+                        {this.renderSuggestions('suggestions')}
```

Comment:
1. Use constants here.
2. Check if using a map would work here:
```suggestion
                        {['recent', 'suggestions'].map(this.renderSuggestions)}
```